.github/workflows/wheels.yml

```diff
@@ -0,0 +1,80 @@
+name: cuDF wheels
+
+on:
+  push:
+    branches:
+      - 'pull-request/[0-9]+'
+  workflow_call:
+    inputs:
+      versioneer-override:
+        type: string
+        default: ''
+      build-tag:
+        type: string
+        default: ''
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
+      build-type:
+        type: string
+        default: nightly
+
+concurrency:
+  group: "cudf-${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  cudf-wheels:
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux.yml@main
+    with:
+      repo: rapidsai/cudf
+
+      build-type: ${{ inputs.build-type || 'pull-request' }}
```

Comment:
Like with last time, noting that we should clean up the pull-request trigger before merging.